packages/syntax-core/src/Tabs/TabButton.module.css

```diff
@@ -0,0 +1,18 @@
+.tab {
```

Comment:
id probably combine your 3 css files to just one, that way you can just reuse some of the styles. alot of them overlap for the most part (at least the classes do)